### ISSUE TITLE
CSV import not working due language selection

### ISSUE THREAD

**Issue by Rodericco** [portfolio-performance/portfolio#3734]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Linux, macOS]
 - Version [e.g. 0.47.0]

**Comment by chirlu** [comment]
The error shouldn’t happen, but the data also looks strange with all those double `""`.

**Comment by Rodericco** [comment]
[0P0001ESJI Historische Daten.csv](https://github.com/portfolio-performance/portfolio/files/13962550/0P0001ESJI.Historische.Daten.csv)
This is the CSV file downloaded from Investing.de, hope it helps guessing what is the problem.

**Comment by flywire** [comment]
Help, Settings, Language shows the date and amount settings for the selected language. I can't import that file with a Language setting of English because it is presented in the wrong format.

**Comment by Rodericco** [comment]
Yes, but when you import, you can choose the format in which the data is going to be readed. That is not a problem for me (the data I use is Deutch). The problem is that I can't import anymore from a Spanish from Spain version of PP. I don't know the reason, but I can import it from Spanish from Cuba, for instance.

**Comment by Nirus2000** [comment]
Plz make a video with screencast software to show us the issue.

**Comment by Rodericco** [comment]
I have trouble because I use Game Bar to screencast and it doesn't show the windows that PP opens. If you have any recommendation, I'll do it.

Anyway, I'll outline the problem:

Language set in Spanish from Spain (es_ES)
Go to the ETF -> Quotes -> Import from CSV -> I choose the CSV from investing.de I attached here

> [0P0001ESJI Historische Daten.csv](https://github.com/portfolio-performance/portfolio/files/13962550/0P0001ESJI.Historische.Daten.csv) This is the CSV file downloaded from Investing.de, hope it helps guessing what is the problem.

And then I get this screen:
![2662792a134369de93473c0d917cc057538b1e2b](https://github.com/portfolio-performance/portfolio/assets/80072186/1b087121-aa07-48e8-b0f3-4d588ae20b36)


Thank you!

**Comment by buchen** [comment]
Thanks @Rodericco for the detailed error description (and, btw, thanks for all your work on the Spanish translation - very much appreciated!!!)

I can reproduce the problem. It also happens in German. Apparently the latest version of the CSV parser behaves slightly different. The CSV parser has become more strict. 


What happens? PP guesses the delimiter based on the language. For Spanish (and German) the default delimiter is a semicolon (;). Then PP tries to parse the file. You notice the quotes around the values. Now, if the value is quoted, then the next character must either be the delimiter (and we configured a semicolon) or the end of the line. But here it is a comma. And then parser fails

```
"16.01.2024","182,950","182,950","182,950","182,950","0,29%"
```

I will fix this, but it will take a couple days as I am busy.


What you can try as a workaround:
* open another CSV file - and could even be an empty file with the file extension ".csv"
* Set the delimiter to comma and save it as a template
* The from the menu import the file using the template (... -> plantillas -> your template)

**Comment by buchen** [comment]
I "fixed" it in such a way that at least the dialog opens and one can pick a different separator char.
Still looking for a better idea to tell the user what could be the problem...
